### PR TITLE
minor cabling tweaks

### PR DESCRIPTION
**Pull request by esaruoho**
this tries to clean up the cabling a bit
@antonhornquist 
please let me know if something like this would be acceptable to you

### CODE REVIEW

**Comment by antonhornquist** [comment]
thanks, i'll have a look

**Comment by esaruoho** [comment]
> thanks, i'll have a look

much appreciated!

**Comment by antonhornquist** [comment]
i took a look at this but it's challenging to identify what's changed in the PR. Pd is a visual language and there's no way to diff visually (i simply opened the differing help patches side by side and could not clearly see the changes). Can you summarize the changes and perhaps include some screenshots to describe the how the cabling is cleaned up? thanks

**Comment by esaruoho** [comment]
@antonhornquist sorry it took me a while to get back to this.
left is before, right is after:
Screenshot 2023-04-02 at 0 32 01


so it adds the github url, straightens some of the lines/patching relationships, etc. hope this screenshot helps.

**Comment by esaruoho** [comment]
@antonhornquist how's the screenshot helping you look at this PR, now?

**Comment by antonhornquist** [comment]
sorry - i'd prefer to keep it as is.
